.travis.yml

```diff
@@ -49,15 +49,14 @@ jobs:
   include:
     - go: "1.11.x"
```

Comment:
Should we drop 1.11 now?

Reply:
We could eventually. I don't think there's much to gain by dropping it now.